dvc/scm/git/__init__.py

```diff
@@ -363,7 +363,12 @@ def get_rev(self):
         return self.repo.git.rev_parse("HEAD")
 
     def resolve_rev(self, rev):
-        return self.repo.git.rev_parse(rev)
+        from git.exc import GitCommandError
+
+        try:
+            return self.repo.git.rev_parse(rev)
+        except GitCommandError as exc:
+            raise RevError(url=self.root_dir, rev=rev) from exc
```

Comment:
It would be awesome if we could tweak `RevError` in this PR to pass a custom message instead of `url, rev`, this will make the error message less verbose, since there's no need for a `url`

Reply:
Not sure if this is out of scope, @efiop ?

Reply:
@mroutis Could you please elaborate? But yeah, maybe let's move it into a separate ticket.